apps/vscode/vscode.py

```diff
@@ -223,6 +223,14 @@ def split_window_vertically():
     def split_window():
         actions.user.vscode("workbench.action.splitEditor")
 
+    def split_number(index: int):
+        """Navigates to a the specified split"""
```

Comment:
Please remove docstrings from implementations of actions.

Reply:
Done